src/Components/Form.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState } from "react";
 import Input from "./Input";
 import TextArea from "./TextArea";
+import Button from "./Button";
 import styled from "styled-components";
-import Logo from "./Logo";
+import { useDispatch } from "react-redux";
```

Comment:
put useDispatch and useSelector on the same line
`import { useSelector, useDispatch } from "react-redux";`